src/main/include/subsystems/FlywheelSubsystem.h

```diff
@@ -23,6 +25,8 @@ class FlywheelSubsystem : public frc2::SubsystemBase
 
 private:
     WPI_TalonFX m_leader{Constants::Flywheel::LeaderID};
+    WPI_TalonFX m_feeder{11};
+    frc::Joystick m_joy{1};

```

Comment:
keep joysticks in RobotContainer and use commands to interact with subsystems as described above